storage/client/client.go

```diff
@@ -1580,9 +1644,36 @@ func (c *StorageClient) RuntimeAccount(ctx context.Context, address staking.Addr
 		return nil, wrapError(err)
 	}
 
+	// Reconcile sdk balance(s) from node with the balance(s) fetched from the db.
+	c.upsertBalances(ch, &a)
+
 	return &a, nil
 }
 
+// Reads node sdk balances from ch and upserts them into acct.Balances, logging a
+// warning if the balances are mismatched.
+func (c *StorageClient) upsertBalances(ch chan *RuntimeSdkBalance, acct *RuntimeAccount) {
+	for nb := range ch {
+		found := false
+		// If the node balance differs from the balance in Nexus, defer to the node balance.
```

Comment:
possibly unpopular opinion, but could we include both in the API :grimacing: 

I think it'd be clearer if the explorer could show a note that it's using the latest balance when it uses the answer from the node. because I see the explanation that we're querying at latest, which can be inconsistent with the latest indexed height

Reply:
consistency is good 👍 

querying at latest indexed height now.